RequireSetup

```diff
@@ -1,4 +1,4 @@
 Increment the below number whenever it is required to run Setup.bat as part of a new commit.
 Our git hooks will detect this file has been updated and automatically run Setup.bat on pull.
 
-80
+81
```

Comment:
Just curious, under what circumstances would it be necessary to modify here?

Reply:
Any changes which require Setup.bat to run, in this case schema was changed so running Setup.bat will ensure schema is regenerated (Setup will delete the schema directory)